test/unit/system/shell_test.py

```diff
@@ -0,0 +1,17 @@
+from pytest import raises
+
+from kiwi.system.shell import Shell
+from kiwi.exceptions import KiwiShellVariableValueError
+
+
+class TestSystemShell:
+    def test_format_to_variable_value(self):
+        assert Shell.format_to_variable_value('text') == 'text'
+        assert Shell.format_to_variable_value(True) == 'true'
+        assert Shell.format_to_variable_value(False) == 'false'
+        assert Shell.format_to_variable_value('42') == '42'
+        assert Shell.format_to_variable_value(0) == '0'
+        assert Shell.format_to_variable_value(42) == '42'
+        assert Shell.format_to_variable_value(None) == ''
```

Comment:
```python
assert Shell.format_to_variable_value(b"42") == "42"
```
:yum:

Reply:
binary stuff at this point, I really hope that this path is never reached :) ...anyway your test passes now ;)